src/config/packages/security.yaml

```diff
@@ -1,39 +1,33 @@
 security:
-    encoders:
-        App\Entity\User:
-            algorithm: auto
+    enable_authenticator_manager: true
+    password_hashers:
+      App\Entity\User:
+        algorithm: auto
 
-    # https://symfony.com/doc/current/security.html#where-do-users-come-from-user-providers
     providers:
-        # used to reload user from session & other features (e.g. switch_user)
         app_user_provider:
             entity:
                 class: App\Entity\User
                 property: username
+
     firewalls:
         dev:
             pattern: ^/(_(profiler|wdt)|css|images|js)/
             security: false
         main:
-            anonymous: true
             lazy: true
             provider: app_user_provider
-            guard:
-                authenticators:
-                    - App\Security\LoginFormAuthenticator
+            form_login:
+              login_path: app_login
+              check_path: app_login
+              enable_csrf: true
+              username_parameter: username
+              password_parameter: password
+            remember_me:
```

Comment:
Thought this might be useful? feel free to dispute it... Unless you think we should add it as a field on login?

Reply:
What does this do? Keeps it saved so it does not ask for login so often?
I think no need to add it as a field on login :)

Reply:
Exactly - keeps you logged in for a week